include/grpc/grpc_security_constants.h

```diff
@@ -49,6 +49,12 @@ typedef enum {
 } grpc_ssl_roots_override_result;
 
 typedef enum {
+  GRPC_GET_SERVER_CREDENTIALS_UNCHANGED,
+  GRPC_GET_SERVER_CREDENTIALS_NEW,
+  GRPC_GET_SERVER_CREDENTIALS_FAIL
+} grpc_get_server_credentials_result;
```

Comment:
Or some `GRPC_SSL_CONFIG_UNCHANGED` etc...

Reply:
> We will want to have the same for clients as well.

I think clients do not need this feature, because clients can just create a new channel when they have new credentials. But if you think this is absolutely necessary, can we add client support in a subsequent work item? It'd be great if the client support doesn't hold up the server support :)

Reply:
Clients do not strictly need this feature and it's ok to not implement for now but the API is there to stay and we have to do the right thing so that it will work for clients as well.
Imagine an application which is both client and server and uses the same SSL cert/key that gets periodically refreshed. It would be clumsy to use a callback for the server but to destroy and recreate the channel for the client.